pyaerocom/aeroval/setup_classes.py

```diff
@@ -297,6 +298,7 @@ class ExperimentInfo(BaseModel):
     public: bool = False
     exp_pi: str = getuser()
     pyaerocom_version: str = __version__
+    creation_date: str = datetime.today().strftime("%Y-%m-%d")
```

Comment:
Please use a json datestamp like `2012-04-23T18:25:43.511Z` please, let the designer decide how it looks on the web.

Reply:
The updates will serialize to a non-timezone aware string like this, if you're okay with it. 
`{'creation_date': '2024-09-19 15:46:36.036175'}`

`datetime.now()` returns a datetime in your local time zone, but it is not timezone-aware.

Reply:
```
creation_date: str = f"{datetime.datetime.now(datetime.timezone.utc):%Y-%m-%dT%H:%M:%S.%fZ"
```

Reply:
Changed